jwcrypto/jwa.py

```diff
@@ -1037,7 +1063,8 @@ class JWA(object):
         'A256CBC-HS512': _A256CbcHs512,
         'A128GCM': _A128Gcm,
         'A192GCM': _A192Gcm,
-        'A256GCM': _A256Gcm
+        'A256GCM': _A256Gcm,
+        'EdDSA': _EdDsa
```

Comment:
Same let's move this up near EcDH so asymmetric signature algorithms are close together.

Reply:
Will do.